recaptcha_enterprise/snippets/src/main/java/recaptcha/MigrateKey.java

```diff
@@ -39,8 +39,8 @@ public static void main(String[] args) throws IOException {
    * to reCAPTCHA Enterprise. For more info, see:
    * https://cloud.google.com/recaptcha-enterprise/docs/migrate-recaptcha
    *
-   * @param projectId: Google Cloud Project Id.
-   * @param recaptchaSiteKey: Specify the site key to migrate.
+   * @param projectId : Google Cloud Project Id.
```

Comment:
nit: to make the variable description less confusing it may make sense to fix casing and avoid abbreviation
```suggestion
   * @param projectId Google Cloud project ID
```